transaction-relayer/src/main.rs

```diff
@@ -268,29 +270,54 @@ fn get_sockets(args: &Args) -> Sockets {
         assert!(!tpu_fwd_ports.contains(&tpu_port));
     }
 
-    let (tpu_p, tpu_quic_sockets): (Vec<_>, Vec<_>) = (0..args.num_tpu_quic_servers)
+    let (tpu_p, tpu_quic_sockets): (Vec<_>, Vec<Vec<_>>) = (0..args.num_tpu_quic_servers)
         .map(|i| {
             let (port, mut sock) = multi_bind_in_range(
                 IpAddr::V4(Ipv4Addr::from([0, 0, 0, 0])),
-                (tpu_ports.start + i, tpu_ports.start + 1 + i),
+                (tpu_ports.start + i, tpu_ports.start + i + 1),
                 1,
             )
             .unwrap();
 
-            (port, sock.pop().unwrap())
+            let quic_config = SocketConfig {
+                reuseaddr: false,
+                reuseport: true,
+            };
+
+            let transactions_quic_sockets = bind_more_with_config(
+                sock.pop().unwrap(),
+                args.num_quic_endpoints.into(),
+                quic_config.clone(),
+            )
+            .unwrap();
+
+            (port, transactions_quic_sockets)
         })
         .unzip();
 
-    let (tpu_fwd_p, tpu_fwd_quic_sockets): (Vec<_>, Vec<_>) = (0..args.num_tpu_fwd_quic_servers)
+    let (tpu_fwd_p, tpu_fwd_quic_sockets): (Vec<_>, Vec<Vec<_>>) = (0..args
```

Comment:
can you add comment on the grouping here?
// this contains a list of tpu_forward_ports and a 2d array of multi-bound quic endpoints, where each outer array maps to the port in the first array